src/main/java/com/kainos/ea/model/JobSpecification.java

```diff
@@ -0,0 +1,45 @@
+package com.kainos.ea.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JobSpecification {
+
+    private int id;
+    private String jobSpecification;
+    private String jobSpecificationLink;
+
+    @JsonCreator
+    public JobSpecification(
+            @JsonProperty("id") int id,
```

Comment:
do you need the ID back? I would imagine you would need the role name back to display on the frontend

Reply:
switched id for job name